package.json

```diff
@@ -398,12 +398,8 @@
     "fp-ts": "^2.3.1",
     "geckodriver": "^1.20.0",
     "getopts": "^2.2.5",
-    "grunt": "1.0.4",
-    "grunt-available-tasks": "^0.6.3",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-peg": "^2.0.1",
```

Comment:
Used to convert `src/plugins/data/common/es_query/kuery/ast/kuery.peg` to js, configured by `tasks/config/peg.js`.